README.md

```diff
@@ -1,24 +1,192 @@
-# README
+<div align="center">
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-Things you may want to cover:
+  <h3><b>Welcome this is Blog App Rails</b></h3>
 
-* Ruby version
+</div>
 
-* System dependencies
 
-* Configuration
+# 📗 Table of Contents
 
-* Database creation
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* Database initialization
 
-* How to run the test suite
+# 📖 [MBudget_app] <a name="about-project"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+> This is a blog app built with Ruby on Rails. It allows users to create an account, create posts, and comment on other posts.
 
-* Deployment instructions
+## 🛠 Built With <a name="built-with"></a>
 
-* ...
++ Ruby
++ Ruby on Rails
++ PostgreSQL
++ Rspec (for testing)
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Users can create an account
+- Users can create posts
+- Users can comment on posts
+- Users can like posts
+- Users can follow other users
+- Users can see posts from users they follow
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> [Live Demo Link](https://blog-app-rails.herokuapp.com/)
```

Comment:
- [ ] Nice work deploying your app to Heroku, kindly check the demo link redirects to the default Heroku page.

Reply:
Hello there 👋,
I have some technical issues with my machine.
This affect the quality of the presentation. I will fix the problem later and redo the presentation, thanks.